Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
```

Comment:
This dockerfile is used to test submission to AI platform, you can see an example on this branch
https://github.com/square/bionic/blob/baxen/ai-platform/bionic/aip/demo.py#L24

We can remove this whole file from the repo if you prefer, but it seems useful to have a reference docker file so that the development team can easily test AIP features in the future

Reply:
Thanks! I agree, it makes a lot of sense to have this reference file here. Maybe we can just add a comment at the top explaining what it's for?